packages/go/analysis/analysis.go

```diff
@@ -127,6 +128,19 @@ func ExpandGroupMembershipPaths(tx graph.Transaction, candidates graph.NodeSet)
 				BranchQuery: func() graph.Criteria {
 					return query.Kind(query.Relationship(), ad.MemberOf)
 				},
+				PathFilter: func(ctx *ops.TraversalContext, segment *graph.PathSegment) bool {
```

Comment:
Having a comment here as to what this is doing might be helpful
Once the ETAC feature flag is removed, that ETAC comment will be removed and it won't be evident that this is specific to ETAC